worker/multi_tenancy.go

```diff
@@ -0,0 +1,90 @@
+// +build !oss
+
+/*
+ * Copyright 2018 Dgraph Labs, Inc. and Contributors
+ *
+ * Licensed under the Dgraph Community License (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ *     https://github.com/dgraph-io/dgraph/blob/master/licenses/DCL.txt
+ */
+
+package worker
+
+import (
+	"context"
+	"time"
+
+	"github.com/dgraph-io/dgraph/conn"
+	"github.com/dgraph-io/dgraph/protos/pb"
+	"github.com/pkg/errors"
+)
+
+func (w *grpcWorker) DeleteNamespace(ctx context.Context,
```

Comment:
Add declaration of all these functions in some file with `+build oss`. Those functions may return not supported error.

Reply:
Thanks. Done!